packages/e2e-tests/fixtures/blocks/core__verse.json

```
@@ -4,7 +4,14 @@
         "name": "core/verse",
         "isValid": true,
         "attributes": {
-            "content": "A <em>verse</em>…<br>And more!"
+            "content": "A <em>verse</em>…<br>And more!",
+            "style": {
+                "spacing": {
+                    "padding": {
+                        "default": 20
```

same as above, this needs to be:

```suggestion
						"top": 20,
						"right": 20,
						"bottom": 20,
						"left": 20
```

---

It applies 20 in the control 🤷‍♂️

---

Yeah, I saw that, but the style mechanism doesn't work that way: it didn't apply the padding style number to the element, did it for you?